### ISSUE TITLE
Extra Double Quote in PostgreSqlDataProvider Generated SQL

### ISSUE THREAD

**Issue by granadacoder** [thepirat000/Audit.NET#623]
**Describe the bug**

Between versions

20.1.5 (no bug)
and 20.1.6 (bug began)

There is a "extra double quote" in the 

PostgreSqlDataProvider

**To Reproduce**
Steps or code to reproduce the behavior.

Setup:

```
            Audit.Core.Configuration.Setup()
                .UseCustomProvider(new CustomPostgreSqlDataProvider(config => config
                    .ConnectionString(AzureSettings.PostgreSqlConnectionString)
                    .Schema("audit")
                    .TableName("MyAuditEventTable")
                    .IdColumnName("MyAuditEventTableKey")
                    .LastUpdatedColumnName("MyPascalCaseUpdateDate")
                    .CustomColumn("MyEventType", ev => ev.EventType)
                    .CustomColumn("MyEnvironUserName", ev => ev.Environment.UserName)
                    .DataColumn("MyEventData", DataType.JSONB)));
            
```

the bug is an extra double quote in the generated sql (see below directly before the WHERE clause)

`SELECT "MyEventData" FROM "audit"."MyAuditEventTable"" WHERE  "MyAuditEventTable"."EventType"='EntityItemSettingCategory'
 `
Note the double “ before the where clause.
 

**Expected behavior**

A working generated sql query   :)

**Libraries (specify the Audit.NET extensions being used including version):**

        <PackageReference Include="Audit.NET" Version="20.1.6" />
        <PackageReference Include="Audit.NET.PostgreSql" Version="20.1.6" />
(and after 20.1.6)

**Target .NET framework:**
For example:

(I don't think this matters but net(core) 6.0.

**Additional context**

I believe the bug is here:

https://github.com/thepirat000/Audit.NET/blob/master/src/Audit.NET.PostgreSql/Providers/PostgreSqlDataProvider.cs#L251

`            var sql = $@"SELECT ""{GetDataColumnName(null)}"" FROM {GetFullTableName(null)}"" {where}";`

Note the same extra double quote before the {where}.

the "GetFullTableName" method... is already quote-surrounding the schema and tablename.

**Comment by thepirat000** [comment]
This was fixed in version 21.0.4, please upgrade your references and re-test.

Thank you!

**Comment by granadacoder** [comment]
Ok.  I see you already did a build.  I will test.  Thank you.
